i2c/sw/sensirion_sw_i2c_gpio.h

```diff
@@ -32,7 +32,17 @@
 #ifndef SENSIRION_SW_I2C_GPIO_H
 #define SENSIRION_SW_I2C_GPIO_H
 
-#include "sensirion_arch_config.h"
+#include "sensirion_common.h"
+#include "sensirion_config.h"
+
+/**
+ * The clock period of the i2c bus in microseconds. Increase this, if your GPIO
+ * ports cannot support a 200 kHz output rate. (2 * 1 / 10usec == 200Khz)
+ *
+ * This is only relevant for the sw-i2c HAL (bit-banging on GPIO pins). The
+ * pulse length is half the clock period, the number should thus be even.
+ */
+#define SENSIRION_I2C_CLOCK_PERIOD_USEC 10
```

Comment:
Why did we now end up with having it defined twice? Accident I guess? It is also defined in `i2c/sw/sensirion_sw_i2c_implementation.c `

Reply:
Ups ... yes that's an accident.